src/audio_input.h

```diff
@@ -14,14 +14,15 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-#ifndef AUDIO_INPUT_H
-#define AUDIO_INPUT_H
+#ifndef SRC_AUDIO_INPUT_H_
+#define SRC_AUDIO_INPUT_H_
 
 #include <functional>
 #include <iostream>
 #include <memory>
 #include <mutex>
 #include <thread>
+#include <utility>
```

Comment:
Does this use the `<utility>` lib?

Reply:
Yes for std::move. Find with cpplint.py.